lib/pages/settings/advanced/maintenance_page.dart

```diff
@@ -78,7 +78,7 @@ class MaintenancePage extends StatelessWidget {
                         isDestructiveAction: true,
                         isDefaultAction: true,
                         style: settingsCardTextStyle.copyWith(
-                          fontSize: 14,
+                          fontSize: 24.6,
```

Comment:
Is that what we use elsewhere? I don't think I've seen a font size like that. Should instead be one of the predefined ones like fontSizeMedium or so